R/getGfortran.R

```diff
@@ -1,4 +1,20 @@
-.getOrUpdateGfortran <- function() {
+#' Download and install Gfortran updates
+#'
+#' @title Download and install Gfortran updates
+#' @return A boolean that represents if the latest version of Gfortran for your OS is installed.
+#' @author Michael Neely
+#' @export
+
+updateGfortran <- function() {
```

Comment:
@mnneely What do you think about this function name?

Reply:
It seems like the underscore is becoming the convention for naming variables, functions, etc. in R?

```suggestion
update_gfortran <- function() {
```

Reply:
Ok 👍

Reply:
Should it have a prefix? lime PMupdate_gfortran?

Reply:
Here's a nice coding convention reference.  As you can tell, I developed things a long time ago. I would not suffix items as op.1 now.  I would call it op_1.
https://www.r-bloggers.com/🖊-r-coding-style-guide/